tools/docker/atest/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM docker.io/library/node:20-alpine3.17 AS ui
 
 WORKDIR /workspace
-COPY console/atest-ui .
+COPY ../../../console/atest-ui .
```

Comment:
I prefer to keep this file in the root directory. For users or contributors, they don't need to find the `Dockerfile` by reading any documents or scripts. They just need to run `docker build` in the root directory.

And the `COPY` command will be very straightforward. For me: `COPY ../../../console/atest-ui` is not clear compared to `COPY console/atest-ui`.

Some tools will try to find the `Dockerfile` in the root directory to build it. Such as [zeabur/zbpack](https://github.com/zeabur/zbpack).

Reply:
got it, I will restore it.